docs/cn_doc/cn-nGQL-tutorial.md

```diff
@@ -0,0 +1,263 @@
+# 查询语言nGQL
```

Comment:
This page is a bit oud-dated (much changed). Maybe we can omit it.

Reply:
Let's comment out this file now.
We'll move back to work on this file after next week